index.html

```diff
@@ -823,7 +823,7 @@
       <li id="step-response-end">Record the time as <a for="PerformanceResourceTiming">responseEnd</a>
       immediately after receiving the last byte of the response.
       <ol>
-        <li>Return to step <a href="#step-connect-start">13</a> if the user agent fails to send the request or receive the entire response, and needs to reopen the connection.
+        <li>Return to step <a href="#step-connect-start">12</a> if the user agent fails to send the request or receive the entire response, and needs to reopen the connection.

```

Comment:
I think current wording is fine. If it doesn't need/want to reopen it'll just continue and set responseEnd on 17?